packages/flutter/test/widgets/image_filter_test.dart

```diff
@@ -31,8 +31,10 @@ void main() {
   });
 
   testWidgets('Image filter - blur with offset', (WidgetTester tester) async {
+    final Key key = GlobalKey();
     await tester.pumpWidget(
       RepaintBoundary(
+        key: key,
```

Comment:
without these changes the golden will sample only the ImageFilter and not the entire screen. The behavior of matchesGoldenFile is that it starts from the element that the finder returns and walks up the tree until it hits a render object that sets isRepaintBoundary to true. Since I made the ImageFilter render object a repaint boundary, it stops sooner and doesn't capture the entire screen.

Add a key to force the screenshots to be the same.